src/main/java/controller/BankController.java

```diff
@@ -0,0 +1,134 @@
+package controller;
+
+import bank.GeneralMember;
+import bank.BankService;
+import bank.SavingsMember;
+import interest.InterestService;
+import util.Appconfig;
+import view.InputView;
+import view.OutputView;
+
+import java.math.BigDecimal;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static util.ErrorMessage.WRONG_FORM;
+
+
+public class BankController {
+
+    private final InputView inputView;
+    private final OutputView outputView;
+
+
+    Appconfig appConfig = new Appconfig();
+    BankService bankService = appConfig.bankService();
+    InterestService interestService = appConfig.interBankService();
+
+
+    public BankController(InputView inputView, OutputView outputView) {
+        this.inputView = inputView;
+        this.outputView = outputView;
+
+    }
+
+    private static final Map<Integer, Runnable> menuOptions = new HashMap<>();
+
+    {
+        menuOptions.put(1, this::createAccount);
```

Comment:
1은 어떤 것을 의미하는 것인가요? 상수를 그대로 사용하지 않고 변수로 선언하여 해당 상수가 어떤 역할을 하고 있는지 나타낼 수 있는 구체적인 변수명을 지어주세요

Reply:
사용자가 입력한 숫자를  나타내는데 이러한 경우에도 상수처리 해주는게 나을까요?

Reply:
예를 들어 createAccount를 하기 위한 숫자가 현재는 1이지만 어떠한 경우로 인해 3번으로 바뀌게 될 경우 코드를 들여다보고 해당 부분을 변경해야 합니다.
하지만 createAccount를 위한 번호라는 변수명을 가진 변수로 해당 상수를 관리하여 적용하면 더 관리하기가 쉽지 않을까요?
```suggestion
        menuOptions.put(CREATE_ACCOUNT_NUMBER, this::createAccount);
```